### PR TITLE
News Item 13 Sept

### PR DESCRIPTION
**Pull request by emmahodcroft**
News item for 13 Sept 2024 - should go live soon if possible.

Took some liberty in spacing and adding headings to make it look nice. Hopefully preview will work.

EB resolutions link won't work until new push to website goes through. This and the newsitem can hopefully be done together.

https://preview-news-item-sept.pathoplexus.org/news/2024-09-13-two-weeks

### CODE REVIEW

**Comment by corneliusroemer** [comment]
Preview works! I added a link to the PR, as well:

Brave Browser 2024-09-13 19 46 16

File: monorepo/website/src/pages/news/2024-09-13-two-weeks.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ & [Github](https://github.com/pathoplexus/pathoplexus), read about our aims & commitments, and also helped spread the word about Pathoplexus!
+
+Here’s a short list of updates from our side, to celebrate two weeks of being live!
+
+### In the media:
+- [Science Magazine’s Jon Cohen covered Pathoplexus](https://www.science.org/content/article/new-scientist-run-virus-database-vows-be-transparently-run-and-simple-use), getting some reactions from the community, and a deep-dive to answer some common questions (29 Aug 2024)
+- [UniVadis’ Roberta Villa gave some history on pathogen data sharing](https://www.univadis.it/index.php/viewarticle/si-dice-villa-genomi-virali-nuova-piattaforma-garantire-2024a1000fx1) and how Pathoplexus might fit into the landscape (Italian, 3 Sept 2024)
+- [BBC World Service’s Roland Pease spoke with Emma Hodcroft on Science in Action](https://www.bbc.co.uk/sounds/play/w3ct5vd7) to find out more about Pathoplexus and what it aims to do (4 Sept 2024)
```
Comment (corneliusroemer) [inline monorepo/website/src/pages/news/2024-09-13-two-weeks.mdx:22]:
```suggestion
- [BBC World Service’s Roland Pease spoke with Emma Hodcroft on Science in Action](https://www.bbc.co.uk/sounds/play/w3ct5vd7) (starting at 8m30s) to find out more about Pathoplexus and what it aims to do (also on [Spotify](https://open.spotify.com/episode/7CjtPdyQfhzsU3jlQE3jmm) and [Apple Podcasts](https://podcasts.apple.com/podcast/concerning-viruses-found-in-fur-farmed-animals/id262026995?i=1000668522189) (4 Sept 2024)
```

**Review by corneliusroemer — APPROVED** [review]
Just added extra links and mentioned the timestamp if you'd like to add - tested and looks great. Clicked on all the links as well :)

File: monorepo/website/src/pages/news/2024-09-13-two-weeks.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<div>
+    <a href="/news">News</a> > <b>Two Weeks of Pathoplexus</b>
+</div>
+
+# Two Weeks of Pathoplexus
+
+*By the Pathoplexus Team - 13 September 2024*
+
+Since Pathoplexus launched on 27 August, we’ve been humbled by the incredibly positive reaction from the pathogen research, bioinformatics, and public health communities. And it’s been a busy two weeks! We’re thrilled so many people have taken the time to explore the website
+ & [Github](https://github.com/pathoplexus/pathoplexus), read about our aims & commitments, and also helped spread the word about Pathoplexus!
```
Comment (theosanderson) [inline monorepo/website/src/pages/news/2024-09-13-two-weeks.mdx:15]:
```suggestion
 & [GitHub](https://github.com/pathoplexus/pathoplexus), read about our aims & commitments, and also helped spread the word about Pathoplexus!
```

**Comment by corneliusroemer** [comment]
LGTM, will merge and start deployment
image

**Pull request merged**